src/doing/list/commands.py

```diff
@@ -61,7 +61,24 @@
     help="Open overview of issues in the web browser.",
     show_envvar=True,
 )
-def list(assignee, author, label, state, type, web):
+@click.option(
+    "--story_points",
+    required=False,
+    default="",
+    type=str,
+    help="Filter on number of story points. Use 'unassigned' to find empty. Use prefix '>' or '<' for greater and less than.",  # noqa
```

Comment:
What about mentioning the >= and <= inequality symbols?
Phrasing suggestion:
`You can use the following inequality symbols as prefixes: '>', '>=', '<' and '<='.`

Reply:
Good one!